LedRGB.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include "Led.h"
+
+class LedRGB {
+  public:
+    LedRGB();
+
+    LedRGB(unsigned redPin, unsigned greenPin, unsigned bluePin);
+    void setRGB(unsigned r = 255, unsigned g = 255, unsigned b = 255);
```

Comment:
not what I what I meant when I said they dont have a default value.
I meant redBrightness, etc. are not initialized.